examples/llm/pretrain/README.md

```diff
@@ -69,4 +69,14 @@ When running the custom_recipe.py file, it will execute the `custom_llama3_8b` r
    python custom_recipe.py "trainer.max_steps=*2"
    ```
 
-These options provide flexibility in customizing your pretraining recipe directly from the command line.
\ No newline at end of file
+These options provide flexibility in customizing your pretraining recipe directly from the command line.
+
+## Multi-datacenter Example
+`multi_datacenter.py` provides a modified Llama 3.1 70B recipe with a few modifications to boost its performance across 2 datacenters.
+1. 2 distributed optimizer instances. This is achieved by setting `num_distributed_optimizer_instances=2` in `nemo.lightning.MegatronStrategy`.
+1. `net_name` as `socket` for the `inter_dp_cp` communication group. This is achieved through the NCCL config file `multi_dc_nccl_communicator_config.yaml` and passing its path to `nemo.lightning.MegatronStrategy`.
+
```

Comment:
Not sure which MCore version the release will use, but `inter_dp_cp` does not exist in lates MCore any more, it's now [inter_ep_dp](https://github.com/NVIDIA/Megatron-LM/blob/main/megatron/core/parallel_state.py#L1245).